systemd/systemd.go

```diff
@@ -600,8 +610,29 @@ func (s *systemd) getUnitStatus(properties []string, unitNames []string) ([]*Uni
 
 	for _, bs := range statusregex.FindAllSubmatch(bs, -1) {
 		if len(bs[0]) == 0 {
+			sts = append(sts, cur)
```

Comment:
nitpick: I would slightly prefer the append to be just before the cur rest below: `cur = &UnitStatus{}`

Reply:
Moved down. Fixed.